CMakeLists.txt

```diff
@@ -511,3 +511,8 @@ else()
     FILE(READ ${PROJECT_BINARY_DIR}/minimal.log GINKGO_LOG_SUMMARY)
 endif()
 MESSAGE(STATUS "${GINKGO_LOG_SUMMARY}")
+
+# make sure no build files get committed accidentally
+if(NOT EXISTS ${CMAKE_CURRENT_BINARY_DIR}/.gitignore)
+    file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/.gitignore "*")
```

Comment:
won't that create a .gitignore in the binary dir that the user has to add to the top-level .gitignore?

Reply:
I think gitignore is slightly more relaxed about dotfiles than bash globs, [the documentation](https://git-scm.com/docs/gitignore) states
> An asterisk "*" matches anything except a slash [...]